sequencer/src/state.rs

```diff
@@ -251,6 +251,21 @@ impl FeeInfo {
         Self { account, amount }
     }
 }
+
+impl Committable for FeeInfo {
+    fn commit(&self) -> Commitment<Self> {
+        let mut comm_bytes = vec![];
+        self.serialize_with_mode(&mut comm_bytes, ark_serialize::Compress::Yes)
+            .unwrap();
```

Comment:
We're still making this a bit awkward to port by serializing here. Better to do the individual fields as `var_size_field`. Serializing the fields themselves is fine because for both `FeeAccount` and `FeeAmount`, the serialization just converts directly to bytes in the intuitive way